### ISSUE TITLE
Transitions between states not showing when changeing a knob

### ISSUE THREAD

**Issue by emattias** [storybookjs/ember-cli-storybook#40]
When changing a knob the transitions between states arent showing.

It seems likee the whole component is reinserted everytume. How can I avoid a full re-render/remount?

**Comment by emattias** [comment]
I got it working :)

**Comment by emattias** [comment]
After further investigation it turns out to not work 😅

**Comment by dbendaou** [comment]
Indeed, that's because whenever you change a knob parameter it re-render the story itself. I don't think there's a way to solve this with the current config. 
I think it require to refact the render method to have a Didupdate hook when changing a know parameter https://github.com/storybookjs/storybook/blob/next/app/ember/src/client/preview/render.ts

**Comment by dbendaou** [comment]
Actually @shilman may have some further informations on it too

**Comment by Alonski** [comment]
@dbendaou @shilman I would love to have this as well :)
